modules/500-upmeter/monitoring/prometheus-rules/alerting.yaml

```diff
@@ -215,3 +215,258 @@
         summary: Smoke-mini is unavailable
         description: |
           Smoke-mini is unavailable. Please check the linked alerts to find the real problem.
+
+- name: d8.upmeter.resources
+  rules:
+    - alert: D8UpmeterProbeGarbageConfigmap
+      expr: |
+        (
+          min_over_time  (count (kube_configmap_info{namespace="d8-upmeter", configmap=~"upmeter-basic-.*"})[5m:])
+          /
+          avg_over_time  (count (kube_pod_labels{namespace="d8-upmeter", label_app="upmeter-agent"})[5m:])
+        ) >= 1
+      labels:
+        severity_level: "9"
+        tier: cluster
+        d8_module: upmeter
+        d8_component: upmeter-agent
+      annotations:
+        plk_protocol_version: "1"
+        plk_markup_format: "markdown"
+        plk_grouped_by__d8_upmeter_resources_garbage: "D8UpmeterProbeGarbage,tier=cluster,prometheus=deckhouse"
+        plk_labels_as_annotations: "configmap"
+        summary: Garbage produced by basic probe is not being cleaned.
+        description: |
+          Probe configmaps found.
+
+          Upmeter agents should clean ConfigMaps produced by control-plane/basic probe. There should not be more
+          configmaps than master nodes (upmeter-agent is a DaemonSet with master nodeSelector). Also, they should be
+          deleted within seconds.
+
+          This might be an indication of a problem with kube-apiserver. Or, possibly, the configmaps were left by old
+          upmeter-agent pods due to Upmeter update.
+
+          1. Check upmeter-agent logs
+
+          `kubectl -n d8-upmeter logs ds/upmeter-agent | jq -rR 'fromjson? | select(.group=="control-plane" and .probe == "basic-functionality") | [.time, .level, .msg] | @tsv'`
+
+          2. Check that control plane is functional.
+
+          3. Delete configmaps manually:
+
+          `kubectl -n d8-upmeter delete cm -l heritage=upmeter`
+
+    - alert: D8UpmeterProbeGarbageDeployment
+      expr: |
+        (
+          min_over_time  (count (kube_deployment_labels{namespace="d8-upmeter",
```

Comment:
Why do you use `_over_time` queries?  `min` and `avg` should be enough for this case, as far as I can tell.